### PR TITLE
#194 :truck: update modules to conform to aissemble naming conventions

### PR DESCRIPTION
**Pull request by habibimoiz**
#194 :truck:  renaming authzforce modules from aiops to aissemble
 - refactored extensions-authzforce
 - refactored extensions-authzforce-attribute
 - extensions-policy-decision-point-service
 - updated velocity templates for generated files.

### CODE REVIEW

File: extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeUtils.java [hunk 1]
```diff
@@ -10,7 +10,6 @@
  * #L%
  */
 
-import com.boozallen.aiops.security.authorization.policy.AiopsAttribute;
```
Comment (aaron-gary) [inline extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeUtils.java:13]: Q: Was this replaced by a corresponding AissembleAttribute, or is is no longer needed?

**Review by aaron-gary — APPROVED** [review]

File: extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeUtils.java (hunk 1 above)
Comment (habibimoiz) [reply extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeUtils.java:13]: The import is no longer needed which is why it was removed.

**Review by ewilkins-csi — CHANGES_REQUESTED** [review]
C: Are we not writing migrations for these?  If we were planning to do that in a follow-on, then we shouldn't merge this to dev

**Comment by habibimoiz** [comment]
> C: Are we not writing migrations for these? If we were planning to do that in a follow-on, then we shouldn't merge this to dev

During my testing, what I found was all of the files that used authzforce were generated classes therefore not requiring any migrations. I tested with both Spark & PySpark implemented projects.

**Comment by habibimoiz** [comment]
Closing pull request. Merging changes in a feature branch instead ([feature-authz-rename](https://github.com/boozallen/aissemble/tree/feature-authz-rename )) to accommodate for the migrations required for #194

**Pull request closed without merging**